caffe-grid/src/main/python/com/yahoo/ml/caffe/DisplayUtils.py

```diff
@@ -36,6 +40,26 @@ def show_df(df, nrows=10):
     html += "</table>"
     return HTML(html)
 
+def show_captions(df, nrows=10):
```

Comment:
This method is very much caption specific. Let's move it to tools/DFConversions.py or tools/Utils.py.